biggraphite/drivers/elasticsearch.py

```diff
@@ -536,8 +534,18 @@ def glob_metric_names(self, glob, start_time=None, end_time=None):
 
         # TODO (t.chataigner) try to move the sort in the ES search and return a generator.
         log.debug(json.dumps(search.to_dict(), default=str))
-        results = [h.name for h in search.execute()]
-        results.sort()
+
+        results = [self._document_to_metric(document) for document in search.execute()]
```

Comment:
How does it work when you have two indexes with the same document ? Which one will be returned ?